cmd/snap/main.go

```diff
@@ -151,8 +149,9 @@ func run() error {
 			return nil
 
 		}
-		if e, ok := err.(*client.Error); ok && e.StatusCode == 401 && e.Kind == client.ErrorKindLoginRequired {
-			return fmt.Errorf(missingLoginHelp)
+		if e, ok := err.(*client.Error); e.Kind == client.ErrorKindLoginRequired {
```

Comment:
hmm, this seems wrong. Either the `ok` should still be checked, or not assigned?